system-addon/test/unit/content-src/components/TopSites.test.js

```diff
@@ -0,0 +1,56 @@
+const React = require("react");
+const {shallow} = require("enzyme");
+const {_unconnected: TopSites, TopSite} = require("content-src/components/TopSites/TopSites");
+
+const DEFAULT_PROPS = {
+  TopSites: {rows: []},
+  dispatch() {}
+};
+
+describe("<TopSites>", () => {
+  it("should render a TopSites element", () => {
+    const el = shallow(<TopSites {...DEFAULT_PROPS} />);
```

Comment:
I'd suggest using enzyme naming conventions here, and calling this "wrapper", since its not the actual element itself.